solutions/observability/apm/configure-kafka-output.md

```diff
@@ -135,7 +135,7 @@ output.kafka:
         message: "ERR"
 ```
 
-This configuration results in topics named _critical-{{version}}_, _error-{{version}}_, and _logs-{{version}}_.
+This configuration results in topics named _critical-{{stack-version}}_, _error-{{stack-version}}_, and _logs-{{stack-version}}_.
```

Comment:
Should this actually not use version variables (is the version the `agent.version`)? 

```suggestion
This configuration results in topics named `critical-{VERSION}`, `error-{VERSION}`, and `logs-{VERSION}`.
```